Tests/MultipeerKitTests/MultipeerKitTests.swift

```diff
@@ -38,7 +38,7 @@ final class MultipeerKitTests: XCTestCase {
 
         mock.receive(TestPayload.self) { payload, sender in
             XCTAssertEqual(payload, tsPayload)
-            XCTAssertEqual(sender.id, mock.localPeerId!)
+            XCTAssertEqual(sender.id, mock.localPeer!.id)
```

Comment:
Just to be thorough, repeat this assertion with the deprecated `localPeerId` property once it's reintroduced as per the previous comment.